### PR TITLE
Create hello-world.json

### PR DESCRIPTION
**Pull request by dfarr**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/foo/pull/1" target="_blank">ReviewNinja

**Comment by dfarr** [comment]
:shipit:  bro

File: hello-world.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "hello": "world"
```
Comment (dfarr) [inline hello-world.json]: test

Comment (dfarr) [inline hello-world.json]: !fix

**Pull request closed without merging**